packages/eas-cli/src/commands/analytics.ts

```diff
@@ -7,7 +7,9 @@ export default class AnalyticsView extends EasCommand {
 
   static override args = [{ name: 'STATUS', options: ['on', 'off'] }];
 
-  protected override requiresAuthentication = false;
+  protected override commandConfiguration: CommandConfiguration = {
+    allowUnauthenticated: true,
```

Comment:
this should work outside of the project context

Reply:
Yep, with this new context concept it'll just work since it doesn't specify the project context.

Reply:
I don't see why `account:view` needs to have `canRunOutsideProject: true` but here it's not necessary?

Reply:
Because the command doesn't declare a context that requires a project (projectId, projectDir), those checks won't be performed and it therefore won't have access to project data.

Reply:
That was my point, `account:view` also does not declare the context that needs projectId